main/release/utils/github.py

```diff
@@ -44,11 +43,25 @@ def get_release_request(self) -> ReleaseRequest:
             release = self._get_release()
             version = self.event['inputs']['version'] if release is None else release['tag_name']
 
-            # tag shall be like X.X.X.BUILD_NUMBER or X.X.X-MX.BUILD_NUMBER or X.X.X+BUILD_NUMBER (SEMVER)
-            version_pattern = re.compile(r'^\d+\.\d+\.\d+(?:-M\d+)?[.+](\d+)$')
+            version_pattern = re.compile(
```

Comment:
same as for your other PR, I think you don't have to teach REGEX in comments, developers should know it. (up to you)

Reply:
I prefer to have documentation. Regex is unintuitive :/